hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMStorage.java

```diff
@@ -32,52 +32,178 @@
 import static org.apache.hadoop.ozone.om.OmUpgradeConfig.ConfigStrings.OZONE_OM_INIT_DEFAULT_LAYOUT_VERSION;
 
 /**
- * OMStorage is responsible for management of the StorageDirectories used by
- * the Ozone Manager.
+ * Ozone Manager VERSION file representation.
+ * On top of what is defined in the base Storage class, this class adds
+ * functionality to hold Ozone Manager related data in its VERSION file.
+ * The additional values stored:
+ * - Ozone Manager ID - a UUID that identifies this Ozone Manager.
+ *                      The value can not be changed once initialized.
+ * - Ozone Manager Node Id - the node id defined for this Ozone manager in the
+ *                           configuration. The value can not be changed after
+ *                           it was set.
+ * - Ozone Manager Certificate Serial Id - the serial id of the Ozone Manager's
+ *                                         SSL certificate if one exists.
  */
 public class OMStorage extends Storage {
 
-  public static final String STORAGE_DIR = "om";
-  public static final String OM_ID = "omUuid";
-  public static final String OM_CERT_SERIAL_ID = "omCertSerialId";
+  static final String ERROR_OM_IS_ALREADY_INITIALIZED =
+      "OM is already initialized.";
+  static final String ERROR_UNEXPECTED_OM_NODE_ID_TEMPLATE =
+      "OM NodeId: {} does not match existing nodeId from VERSION file: {}";
```

Comment:
This is being passed to `String#format`, which uses `%` as the encoding, so the `{}` instances are not replaced. Intellij has a warning for this. `testValidateOrPersistOmNodeIdThrowsWithDifferentNodeIdAsInFile` makes the same mistake, and only checks for substring matches, so it passes.

Reply:
Oh... my bad, I don't know what I was doing here :D Fixing it.